src/GraphQL/ResolveFieldContext/ResolveFieldContextExtensions.cs

```diff
@@ -107,6 +107,44 @@ public static IResolveFieldContext<TSourceType> As<TSourceType>(this IResolveFie
 
         private static readonly char[] _separators = { '.' };
 
+        /// <summary>
+        /// Method to get value by path (key1.key2.keyN) from input extensions dictionary.
+        /// </summary>
+        /// <param name="context">Context with dictionary of extra information supplied with the GraphQL request.</param>
+        /// <param name="path">Path to value in key1.key2.keyN format.</param>
+        /// <returns>Value, if any exists on the specified path, otherwise <see langword="null"/>.</returns>
+        public static object? GetInputExtension(this IResolveFieldContext context, string path)
+        {
+            if (context == null)
+                throw new ArgumentNullException(nameof(context));
+
+            if (context.InputExtensions == null || context.InputExtensions.Count == 0)
+                return null;
+
+            // actually context.InputExtensions is of type GraphQL.Inputs
+            if (context.InputExtensions is not IDictionary<string, object?> values || values.Count == 0)
+                return null;
+
```

Comment:
remove

Reply:
does not compile

Reply:
Well you can’t literally remove it. I meant remove the cast.